src/FsAutoComplete.Core/FakeSupport.fs

```diff
@@ -2,8 +2,96 @@ namespace FsAutoComplete
 
 open Fake.Runtime
 open Fake.Runtime.Trace
+open System
+open System.IO
+open System.IO.Compression
+open System.Threading.Tasks
+open System.Net
+open Newtonsoft.Json.Linq
 
 module FakeSupport =
+  let private compatibleFakeVersion = "5.15.0"
+
+  let private fakeDownloadUri version =
+    sprintf "https://github.com/fsharp/FAKE/releases/download/%s/fake-dotnetcore-portable.zip" version
+    |> Uri
+
+  let downloadAndExtract (uri:Uri) directory = async {
+    use client = new WebClient()
+    client.Headers.Add("user-agent", "Ionide")
```

Comment:
Do we use a common user-agent already somewhere?

Reply:
No, but that's fine. It's actually first place where we download something in FSAC